src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,38 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+    private int side;
+    private int base;
+
+    public RightTriangle(int side, int base, String color) {
+        this.side = side;
+        this.base = base;
+        this.setColor(color);
+    }
+
+    public int getSide() {
+        return side;
+    }
+
+    public int getBase() {
+        return base;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Drawing a righttrianangle");
```

Comment:
The message inside the `System.out.println` should be in English and provide more details about the figure, including its properties and color.